apps/arm/include/TurntableConfig.h

```diff
@@ -0,0 +1,51 @@
+#pragma once
+
+#include "AEAT6012.h"
+#include "ActuatorControllerManager.h"
+#include "Current.h"
+#include "DCMotor.h"
+#include "LookupTable.h"
+#include "OpenLoop.h"
+#include "PID.h"
+#include "Position.h"
+#include "Velocity.h"
+#include "hw_bridge.h"
+
+namespace Turntable {
+
+namespace Internal {
+
+static Encoder::AEAT6012 encoder({TRNTBL_ENC_SPI_CLK, TRNTBL_ENC_SPI_MISO, NC, 0});
+
+static Actuator::DCMotor motor(MTR_PWM_TRNTBL, MTR_DIR_TRNTBL, false);
+
+// static Sensor::CurrentSensor currentSensor(TRNTBL_CRNT_SNS_SPI_CLK, TRNTBL_CRNT_SNS_SPI_MISO,
+// TRNTBL_CRNT_SNS_SPI_CS);
+// TODO: Add once current sensor driver works
+
+static PID::PID velPID({1, 0, 0, -1, 1, 0, false, false});
+static PID::PID posPID({1, 0, 0, -1, 1, 0, false, false});
+static PID::PID curPID({1, 0, 0, -1, 1, 0, false, false});
+
+constexpr uint8_t MAXCURRENT = 53;
+constexpr float MAXDEGPERSEC =
```

Comment:
For consistency, prefix these with the sensor or actuator name

Reply:
Also should max current be a float?

Reply:
it gets implicitly cast to float. since all uint8t's can be represented as floats